base/applications/mscutils/eventvwr/lang/cs-CZ.rc

```diff
@@ -118,9 +118,9 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "&Zobrazovaný název:", IDC_STATIC, 7, 9, 60, 8
     EDITTEXT IDC_DISPLAYNAME, 67, 7, 178, 12, ES_LEFT | ES_AUTOHSCROLL | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
-    LTEXT "&Název protokolu:", IDC_STATIC, 7, 25, 60, 8
+    LTEXT "&Název protokolu:", IDC_STATIC, 7, 25, 60, 8//FIXME: accelerator collision &N
```

Comment:
Please put a separating space before the "//FIXME" part. (For all places where you added these)

Reply:
I did that in the past with the spaces. I intentionally stopped doing it with a space, because in several occassions I had to spoon-feed the translators afterwards to strip that space as well when stripping the comment. They do love to leave that EOL-whitespace existing. So I feel that style to be the better solution now. Keep in mind that those are temporary comments!